src/index.html

```diff
@@ -13,6 +13,78 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section aria-label="Star holder">
```

Comment:
The `section` tag without a heading could be confusing for screen readers. Consider using a `div` tag instead or adding a heading to the section.